packages/components/tabs/src/tab-group.scss

```diff
@@ -125,8 +120,7 @@
 }
 
 [part='container'] {
-  background: var(--_tablist-background);
```

Comment:
The background is missing:

<img width="1145" alt="image" src="https://github.com/user-attachments/assets/3b3a14ce-daa2-42f4-ac92-34272b203b8b" />